Documentation/HowToUseWCFInContainers.md

```diff
@@ -0,0 +1,143 @@
+This document describes how to get started with WCF for .NET Core in containers. It will guide you through creating a sample WCF client & server apps, creating a Windows and Linux Docker containers and share additional details on how to install client certificates in Windows containers.
+
+Prerequisites:
+- Windows Server 2016 or Windows 10 with "Containers" feature enabled
+- Latest [Docker](https://www.docker.com) version installed
+- If running Docker inside a VM, make sure you [enabled nested virtualization for the VM](https://blogs.technet.microsoft.com/virtualization/2015/10/13/windows-insider-preview-nested-virtualization/) 
+
+
+
+### 1. Sample app
+These are the steps for creating a sample WCF service and client. If you already have a sample WCF service app (or even host it in a container as described in an excellent blog post by Jeffrey T. Fritz at https://blogs.msdn.microsoft.com/webdev/2017/02/20/lets-try-wcf-self-hosted-services-in-a-container/), and a corresponding .NET Core WCF client project then you may skip the sample app creation and proceed to step 2. 
```

Comment:
Is service in container something you have tried? People may expect, for example, cert auth to work when the service is hosted in container after reading this doc. If we did not get a chance to validate scenarios with both client and service in containers, I would skip mentioning the service container blog for now.